CHANGELOG.md

```diff
@@ -26,6 +26,7 @@
 * [ENHANCEMENT] Clarify which S3 signature versions are supported in the error "unsupported signature version". #3376
 * [ENHANCEMENT] Store-gateway: improved index header reading performance. #3393
 * [ENHANCEMENT] Store-gateway: improved performance of series matching. #3391
+* [ENHANCEMENT] Move the validation of incoming series before the Distributor's forwarding functionality, so that we don't forward invalid series. #3386
```

Comment:
```suggestion
* [ENHANCEMENT] Move the validation of incoming series before the distributor's forwarding functionality, so that we don't forward invalid series. #3386
```